### DISCUSSION TITLE
error JSON Schema validation failed

### DISCUSSION THREAD

**Discussion by tridoxx** [GetDKAN/dkan#3850] [General]
hi how i can verifi about this error?, when i used this tools show is validated https://dashboard.data.gov/validate 

error: 
![image](https://user-images.githubusercontent.com/22879539/195398426-2c24e5f4-d247-416c-9ca4-df85d7e65ec6.png)

tool:

![image](https://user-images.githubusercontent.com/22879539/195398045-f05d69a9-8eb7-4965-9a50-398c5711772b.png)

and this is my json

`{
    "@context": "https://project-open-data.cio.gov/v1.1/schema/catalog.jsonld",
    "@id": "http://dkan/data.json",
    "@type": "dcat:Catalog",
    "conformsTo": "https://project-open-data.cio.gov/v1.1/schema",
    "describedBy": "https://project-open-data.cio.gov/v1.1/schema/catalog.json",
    "dataset": [
        {
            "@type": "dcat:Dataset",
            "medataAuthor": "Secretaría de Seguridad y convivencia - Sistema de Información para la Seguridad y la Convivencia SISC",
            "medataGranularity": "Municipal",
            "accessLevel": "public",
            "accrualPeriodicity": "R/P1M",

            "description": "Casos que ante una medida de protección cautelar presentan nuevos eventos.",
            "distribution": [
                {
                    "@type": "dcat:Distribution",
                    "downloadURL": "http://3.20.221.84/reincidencia_violencia_intrafamiliar.csv",
                    "mediaType": "text/csv",
                    "format": "csv",
                    "title": "Reincidencia violencia intrafamiliar",
                    "medataEncoding": "utf-8",
                    "medataDelimiter": ";",
                    "describedByType": "application/schema+json",
                    "describedBy": "http://3.20.221.84/Modelo_Unificado_Seguridad.json"
                }
            ],
            "identifier": "Reincidencia violencia intrafamiliar",
            "issued": "2018-06-01",
            "modified": "2022-10-01",
            "license": "https://opendefinition.org/licenses/cc-by-sa/",
            "publisher": {
                "@type": "org:Organization",
                "name": "Seguridad"
            },
            "theme": [
                
                    "Seguridad"
                
            ],
            "temporal": "2007-02-26T00:00:00Z/2018-10-24T00:00:00Z",
            "conformsTo": "http://www.mintic.gov.co",
            "language": [
                    "es-CO"
                ]
            ,
            "title": "Reincidencia violencia intrafamiliar",
            "keyword": [
                "Medellín",
                "Seguridad y convivencia",
                "criminalidad",
                "operatividad"
            ]
        }
    ]
}`

**Comment by tridoxx** [comment]
searching i found that some fields, are not permit so y create it, again with some fields less, and work example `{
    "@context": "https:\/\/project-open-data.cio.gov\/v1.1\/schema\/catalog.jsonld",
    "@id": "http:\/\/dkan\/data.json",
    "@type": "dcat:Catalog",
    "conformsTo": "https:\/\/project-open-data.cio.gov\/v1.1\/schema",
    "describedBy": "https:\/\/project-open-data.cio.gov\/v1.1\/schema\/catalog.json",
    "dataset": [
        {
            "@type": "dcat:Dataset",
            "accessLevel": "public",
            "contactPoint": {
                "fn": "luz Ester Alzate Arias",
                "hasEmail": "mailto:medata@medellin.gov.co"
            },
            "description": ".Casos que ante una medida de protección cautelar presentan nuevos eventos.",
            "distribution": [
                {
                    "@type": "dcat:Distribution",
                    "downloadURL": "http:\/\/3.20.221.84\/reincidencia_violencia_intrafamiliar.csv",
                    "mediaType": "text\/csv",
                    "format": "csv",
                    "title": "Reincidencia violencia intrafamiliar",
                    "describedBy": "http:\/\/3.20.221.84\/Modelo_Unificado_Seguridad.json"
                }
            ],
            "identifier": "Reincidencia violencia intrafamiliar",
            "issued": "2018-06-01",
            "modified": "2022-10-01",
            "keyword": [
                "Medellín",
                "Seguridad y convivencia",
                "criminalidad",
                "operatividad"
            ],
            "license": "https:\/\/opendefinition.org\/licenses\/cc-by-sa\/",
            "modified": "2016-04-21",
            "publisher": {
                "@type": "org:Organization",
                "name": "Seguridad"
            },
            "spatial": "Pittsburgh, PA",
            "temporal": "2016\/P1Y",
            "theme": [
                "Seguridad"
            ],
            "title": "Reincidencia violencia intrafamiliar"
        }
    ]
}`

**Comment by janette** [comment]
@tridoxx If you want to customize the dataset.json schema, you must copy ALL schema files from `schema/collections` and put them in your project `src/schema/collections` directory, then edit the dataset.json and dataset.ui.json

**Reply by tridoxx** [reply]
> Comment options
> ###
> [ ](/janette)

okay i will do, ty. :3
